kernel/kernel-config-4.9.x

```diff
@@ -157,12 +157,13 @@ CONFIG_MEMCG=y
 # CONFIG_BLK_CGROUP is not set
 CONFIG_CGROUP_SCHED=y
 CONFIG_FAIR_GROUP_SCHED=y
-# CONFIG_CFS_BANDWIDTH is not set
+CONFIG_CFS_BANDWIDTH=y
 # CONFIG_RT_GROUP_SCHED is not set
 # CONFIG_CGROUP_PIDS is not set
 CONFIG_CGROUP_FREEZER=y
 # CONFIG_CGROUP_HUGETLB is not set
-# CONFIG_CPUSETS is not set
+CONFIG_CPUSETS=y
+CONFIG_PROC_PID_CPUSET=y
```

Comment:
Just checking - do we need/use this PID_CPUSET fs - it appears to be marked 'legacy' in the Kconfig, and I think it is superseded by the cgroups fs:
https://github.com/clearcontainers/linux/blob/master/init/Kconfig#L821
https://github.com/clearcontainers/linux/blob/master/Documentation/cgroup-v1/cpusets.txt#L769

Reply:
@grahamwhaley ahh you are right, lemme remove PID_CPUSET